dev-ruby/excon/excon-0.78.1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
There is now also https://github.com/gentoo/gentoo/pull/21399 for this package, and I've reviewed that version as well.

Reply:
Yes sure but this ebuild got the tests working and the this version of vagrant_cloud-3.0.2 ask for excon-0.73.0